packages/inspection-capture-web/src/PhotoCapture/PhotoCaptureHUD/PhotoCaptureHUDButtons/PhotoCaptureHUDButtons.tsx

```diff
@@ -1,111 +1,60 @@
 import { Icon, TakePictureButton } from '@monkvision/common-ui-web';
 import { useInteractiveStatus } from '@monkvision/common';
 import { useCaptureHUDButtonsStyles } from './hooks';
-
-/**
- * Props of the PhotoCaptureHUDButtons component.
- */
-export interface PhotoCaptureHUDButtonsProps {
-  /**
-   * URI of the picture displayed in the gallery button icon. Usually, this is the last picture taken by the user. If no
-   * picture is provided, a gallery icon will be displayed instead.
-   */
-  galleryPreview?: string;
-  /**
-   * Callback called when the user clicks on the take picture button.
-   */
-  onTakePicture?: () => void;
-  /**
-   * Callback called when the user clicks on the gallery button.
-   */
-  onOpenGallery?: () => void;
-  /**
-   * Callback called when the user clicks on the close button. If this callback is not provided, the close button will
-   * not be displayed on the screen.
-   */
-  onClose?: () => void;
-  /**
-   * Boolean indicating if the gallery button is disabled.
-   *
-   * @default false
-   */
-  galleryDisabled?: boolean;
-  /**
-   * Boolean indicating if the take picture button is disabled.
-   *
-   * @default false
-   */
-  takePictureDisabled?: boolean;
-  /**
-   * Boolean indicating if the close button is disabled.
-   *
-   * @default false
-   */
-  closeDisabled?: boolean;
-  /**
-   * Boolean indicating if the close button should be displayed in the HUD on top of the Camera preview.
-   *
-   * @default false
-   */
-  showCloseButton?: boolean;
-  /**
-   * Boolean indicating if the little notification badge on top of the gallery button should be displayed.
-   *
-   * @default false
-   */
-  showGalleryBadge?: boolean;
-  /**
-   * Total number of sights to retake
-   *
-   * @default 0
-   */
-  retakeCount?: number;
-}
+import { PhotoCaptureHUDButtonsProps } from './PhotoCaptureHUDButtons.model';
 
 /**
  * Components implementing the main buttons of the PhotoCapture Camera HUD. This component implements 3 buttons :
  * - A take picture button
  * - A gallery button
- * - A close button (only displayed if the `onClose` callback is defined)
+ * - A action button (only displayed if the `onAction` callback is defined)
```

Comment:
An action button...

Reply:
this change had been removed. Previous iteration required to keep the approve aka validate key on the right side above capture icon. where commonly close icon is present.